controllers/metallb_controller.go

```diff
@@ -42,12 +43,21 @@ const defaultMetallbCrName = "metallb"
 // MetallbReconciler reconciles a Metallb object
 type MetallbReconciler struct {
 	client.Client
-	Log    logr.Logger
-	Scheme *runtime.Scheme
+	Log          logr.Logger
+	Scheme       *runtime.Scheme
+	PlatformInfo platform.PlatformInfo
 }
 
 var ManifestPath = "./bindata/deployment"
 
+var TestIsOpenShift = false
```

Comment:
Let's move this in the e2e.go file, where we use it.
The variable can be local to the package at that point,